src/simpleTrianglePatterns/simpleTrianglePattern2.java

```diff
@@ -7,12 +7,12 @@ public static void main(String[] args){
         Scanner userInputScanner = new Scanner(System.in);
         System.out.print("Enter a number:");
         try{
-            int n = userInputScanner.nextInt();
+            int number = userInputScanner.nextInt();
```

Comment:
Whats the difference between this number and an another number? Its still not meaningful.

Reply:
same

Reply:
same